helix-core/src/movement.rs

```diff
@@ -192,11 +204,11 @@ enum Category {
 fn categorize(ch: char) -> Category {
     if ch == '\n' {
         Category::Eol
-    } else if ch.is_ascii_whitespace() {
```

Comment:
I think this could simply use char.is_whitespace, which is included in stdlib

Reply:
Alternatively we can call get_general_category once then switch based on the result

Reply:
It breaks with unicode space, i.e. '　' in Japanese.

Reply:
is_ascii_whitespace does, but is_whitespace works with unicode: https://doc.rust-lang.org/std/primitive.char.html#method.is_whitespace

Reply:
Gotcha

Reply:
Would it be worth using it over the one in the match?